### DISCUSSION TITLE
Code Interpreter效果很差，是什么原因？

### DISCUSSION THREAD

**Discussion by Sylar-W** [zai-org/ChatGLM3#563] [Bad Case]
基本上画啥都画不出来，爱心偶尔也画不出来
![image](https://github.com/THUDM/ChatGLM3/assets/126435040/a81fb48a-3680-4345-9fc1-2fee9bb07c08)

**Comment by zRzRzRzRzRzRzR** [comment]
这个功能基本还在玩具阶段，这个报错代表没法正常执行，生成的代码质量不够，这是很有可能的
